### PR TITLE
Capistrano cleanup

### PR DESCRIPTION
**Pull request by mcritchlow**
### What does this PR do?
- Adds a brief reference to deploying a tag/branch (Capistrano)
- Add additional Capistrano config TODO's for migration

@ucsdlib/developers - please review

### CODE REVIEW

**Review by lsitu — APPROVED** [review]
👍

File: gitlab_migration.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Go to Settings -> CI/CD:
 
 Examples:
 ```
-/gitlab ucsdlibrary/apbatch run staging_deploy
+/gitlab ucsdlibrary/apbatch run staging_deploy 1.5.0
```
Comment (VivianChu) [inline gitlab_migration.md:78]: @mcritchlow - what's that number 1.5.0? Is it a tag? Thanks

Comment (mcritchlow) [reply gitlab_migration.md:78]: @VivianChu - yep that's a tag. Should I add a note about that? Perhaps so, since you needed to ask :)

Comment (mcritchlow) [reply gitlab_migration.md:78]: @VivianChu - I've updated the branch to try and clarify. Can you take another look when you have a chance and let me know if that's better?

**Pull request merged**